Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -22257,5 +22252,377 @@
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "las próximas dos décadas",
+        "Start": 39,
+        "End": 62,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(2020-01-01,2040-01-01,P20Y)",
+              "type": "daterange",
+              "start": "2020-01-01",
+              "end": "2040-01-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Esos acuerdos han proliferado en el último decenio.",
+    "Context": {
+      "ReferenceDateTime": "2018-07-02T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "el último decenio",
+        "Start": 33,
+        "End": 49,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(2000-01-01,2010-01-01,P10Y)",
+              "type": "daterange",
+              "start": "2000-01-01",
+              "end": "2010-01-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Se informará a la Junta semestralmente de los avances en la aplicación",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "semestralmente",
+        "Start": 24,
+        "End": 37,
+        "TypeName": "datetimeV2.set",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "P0.5Y",
+              "type": "set",
+              "value": "not resolved"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Tengo muchas ganancias en este escolar año.",
+    "Context": {
+      "ReferenceDateTime": "2019-06-18T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "este escolar año",
+        "Start": 26,
+        "End": 41,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "SY2019",
+              "type": "daterange",
+              "value": "not resolved"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Tengo muchas ganancias en el último fiscal año.",
+    "Context": {
+      "ReferenceDateTime": "2019-06-18T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "el último fiscal año",
+        "Start": 26,
+        "End": 45,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "FY2018",
+              "type": "daterange",
+              "value": "not resolved"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Tengo muchas ganancias en este calendario año.",
```

Comment:
Do we have cases for the other order? As far as I'm aware both should be supported.
"Se espera que la transacción se cierre este año calendario."

Reply:
yes the other order was already supported ("Tengo muchas ganancias este año calendario"), added also new case